src/retrace-server-interact

```diff
@@ -123,7 +123,20 @@ if __name__ == "__main__":
             if task.has_vmlinux():
                 vmlinux = task.get_vmlinux()
             else:
-                vmlinux = task.prepare_debuginfo(vmcore, kernelver=kernelver)
+                if self.has_status() and \
+                   self.get_status() not in [STATUS_SUCCESS, STATUS_FAIL]:
+                    sys.stderr.write("Task '%s' still in progress or hung (status = %d), "
+                                     "please wait for task to complete.\n" %
+                                     (task.get_taskid(), task.get_status()))
+                    sys.stderr.write("If you suspect task is hung and will never "
+                                     "complete, try forcing failure with "
+                                     "retrace-server-interact, and restart the task.")
+                    exit(1)
```

Comment:
Use `sys.exit` instead, please. There are some tricky issues with plain `exit`.

Reply:
do you want me to add a patch to replace exit() with sys.exit() for all retrace-* commands?

Reply:
Even though it's slightly out of scope of the PR, it's still refactoring and it's a minor change, so I won't mind it. Go ahead.